### ISSUE TITLE
Strange problem on opensuse-tumbleweed (Macro invokation is cancelled by decode_macro_limit)

### ISSUE THREAD

**Issue by thanosz** [akinomyoga/ble.sh#105]
**ble version**:  0.4.2-devel2+267baf2
**Bash version**: 5.1.4(1)-release

Hi! I have noticed that it does not work in opensuse-tumbleweed. When I press '/' it seems it is processing and finally fails with "Macro invocation is cancelled bu decode_macro_limit" displayed at the top.

Please see attached video

https://user-images.githubusercontent.com/8742206/117423143-0f4a9c80-af29-11eb-9551-ded25e44adb1.mp4

**Comment by akinomyoga** [comment]
Thank you for the report! I've installed OpenSUSE Tumbleweed in my VirtualBox and confirmed the problem. I have fixed it in commit ea05fc5. It was caused because the readline macro `"\eOo": "/"` defined in `/etc/inputrc.keys:565,744` was a duplicate of a `ble.sh` internal translation so has been resulted in a loop macro `"/": "/"`. I have modified `ble.sh` to ignore duplicate macros.

Can you check if it works in the latest version? You can update `ble.sh` with `ble-update` in a `ble.sh` session (for which you need `git` and `make`):

```bash
$ sudo zypper install git make # if you haven't yet installed git or make
$ ble-update
```

By the way, `vi-replace` in `vi_imap`, and `vi-editing-mode` in `vi_nmap` are also supported in the latest version; these supports have been added after #89.

**Comment by thanosz** [comment]
Great work @akinomyoga ! I confirm that is fixed

**Comment by akinomyoga** [comment]
Great. Thank you for the check!
